config/gulp/file-prep.js

```diff
@@ -0,0 +1,255 @@
+ */
+function createDir(directoryPath, foldersDeep) {
+  var dp = filePaths.base;
+
+  //if this directory does not exist, create it
+  if (!fs.existsSync(directoryPath)) {
+    //split directory folders
+    var subdir = directoryPath.split("/").slice(foldersDeep);
+    //create parent directories first
+    for (let i = 0; i < subdir.length; i++) {
+      for (let j = i; j <= i; j++) {
+        dp = dp + subdir[j] + "/";
+        console.log("i: " + i + " , j: " + j + "\n" + dp);
+        //check if subfolder exists
+        if (fs.existsSync(dp)) {
+          continue;
+        }
+        // create folder
+        else {
+          fs.mkdirSync(dp, (err) => {
+            "Error creating subdirectory [" + subdir[i] + "]\n",
+              "Error message: " + err.message;
+          });
+        }
+      }
+    }
+  }
+}
+
+/**
+ * Checks the file extension and returns a string value of file or image
+ * @param {String} extension - file name extension (.pdf, .png, etc...)
+ * @returns a string value of image or file
+ */
+function fileType(extension) {
+  if (fileRegex.test(extension)) return "file";
+  if (imageRegex.test(extension)) return "image";
+}
+
+/**
+ * Normalizes filename by removing unnecessary characters
+ * @param {string} origfilename - filename of original uploaded file
+ * @example 
+ * File Name.pdf returns file-name.pdf
+ * @returns filename in string format
+ */
+function cleanFileName(origfilename) {
+  return origfilename
+    .toLowerCase()
+    .replace(/[ &$_#!?.]/g, "-")
+    .replace(/-+/g, "-") // multiple dashes to a single dash
+    .replace(/-(png|jpg|jpeg|pdf|doc|docx|ppt|pptx|pptm|xls|xlsx)/g, ".$1") // remove trailing dashes
+    .replace(/\.jpeg$/g, ".jpg") // .jpeg to .jpg
+    .replace(/-\d{2,4}x\d{2,4}(?=\.jpg)/g, "") // strip trailing dimensions
+    .replace(/^\d{2,4}-*x-*\d{2,4}-*/g, "") // strip leading dimensions
+    .replace(/-\./g, ".") // remove leading dashes
+    .replace(/^-/g, "") // removes dashes from start of filename
+    .toLowerCase();
+}
+
+/**
+ * removes files in content/images/_inbox directories
+ * keeps _inbox/__add jpg and png files to this folder__
+ */
+function cleanInbox() {
+  return del([
+    "content/uploads/_inbox/**",
+    "!content/uploads/_inbox",
+    "!content/uploads/_inbox/__add jpg and png files to this folder__",
+  ]);
+}
+
+
+/**
+ * Creates a timestamp for the yml file
+ * @returns date in string format 2023-01-18 14:05:46 -0400
+ */
+function get_curr_date() {
+  var d = new Date();
+  var month = d.getMonth() + 1;
+  var day = d.getDate();
+  var output =
+    d.getFullYear() +
+    "-" +
+    (month < 10 ? "0" : "") +
+    month +
+    "-" +
+    (day < 10 ? "0" : "") +
+    day +
+    " " +
+    (d.getHours() < 10 ? "0" : "") +
+    d.getHours() +
+    ":" +
+    (d.getMinutes() < 10 ? "0" : "") +
+    d.getMinutes() +
+    ":" +
+    (d.getSeconds() < 10 ? "0" : "") +
+    d.getSeconds() +
+    " -0400";
+  return output;
+}
```

Comment:
Got it, replaced with `new Date().toISOString()`.

Reply:
@nick-mon1 the TODO and note on future reference was for a new issue in the future. Using `toISOString()` makes the function comment invalid:

```
/**
 * Creates a timestamp for the yml file
 * @returns date in string format 2023-01-18 14:05:46 -0400
 */
```

Since this now returns `'2023-01-25T22:04:42.330Z'` instead of the format `2023-01-18 14:05:46 -0400`.

Does this change have any effect on how we use this data?